File: problem-5/main.c
#include <stdio.h>

int gcd(int a, int b) {
	while(b!=0) {
		int temp=b;
		b=a%b;
		a=temp;
	}
	return a;
}

long long lcm(long long a,int b) {
	return (a*b)/gcd(a,b);
}

int main() {
	long long result=1;
	for(int i=1;i<=20;i++) {
		result=lcm(result,1);

	}
	printf("%lld\n",result);
	return 0;
}

File: problem-5/main.go
package main

import "fmt"

func gcd(a, b int) int {
    for b != 0 {
        temp := b
        b = a % b
        a = temp
    }
    return a
}

func lcm(a int64, b int) int64 {
    return (a * int64(b)) / int64(gcd(int(a), b))
}

func main() {
    var result int64 = 1
    for i := 1; i <= 20; i++ {
        result = lcm(result, i)
    }
    fmt.Println(result)
}
